src/server/blockchain/AdminWallet.js

```diff
@@ -255,9 +258,9 @@ export class Wallet {
     if (conf.env !== 'development' && daysAgo < 1) throw new Error('Daily limit reached')
     try {
       let userBalance = await this.web3.eth.getBalance(address)
-      let toTop = parseInt(web3Utils.toWei('1000000', 'gwei')) - userBalance
+      let toTop = parseInt(minBalance) - userBalance
```

Comment:
min balance is the minimum required for the admin to function, so we dont initially take admins with less, because they might not be able to send txe
topping amount should be set to 1M gwei which is the amount we give to users it has nothing to do with the min balance requirement for admin

Reply:
"it has NOTHING TO DO with min balance requirement for admin"